packages/maskbook/src/plugins/Furucombo/UI/InvestmentsView.tsx

```diff
@@ -199,6 +200,7 @@ export function InvestmentsView(props: any) {
                 <TableHead>
                     <TableRow className={classes.head}>
                         {headCells.map((headCell) => {
+                            const label: any = `plugin_furucombo_${headCell.label}`
```

Comment:
we locale linter not supported dynamic string.

plz use hard-coded string, eg: `t('plugin_furucombo_tab_pool')`

https://github.com/DimensionDev/Maskbook/blob/develop/packages/scripts/src/locale-kit/ast.ts
```
t('...')
```

Reply:
Noted. I'll fix it.

Reply:
Done. Removed `any` across the entire project